components/faq.tsx

```diff
@@ -11,8 +11,8 @@ function FaqContainer({ title, text }: FaqContainerType) {
 
 export default function Faq() {
   return (
-    <div>
-      <div className="bg-black py-14 px-44 text-white my-8">
+    <>
```

Comment:
This looks incorrect.

Reply:
I just changed the div tag to an empty tag there